test_data/asm/secondary_block_machine_add2.asm

```diff
@@ -18,8 +18,9 @@ machine Main {
     // The input column needs to be constant for the entire block
     col witness add_two_input;
 
+    col fixed _first_step = [1] + [0]*;
     // Because constraints are not cyclic, we need to explicitly constrain the first state
-    first_step * (add_two_state - add_two_input) = 0;
```

Comment:
The column `first_step` is introduced by the vm-to-constrained transformation. We should find a way that it does not clash with existing columns (and also combine equal columns with an optimizer).

Reply:
Another way would be to provide `first_step` column in the std library.